### ISSUE TITLE
Question about ImageNet16120

### ISSUE THREAD

**Issue by nukeyyou** [D-X-Y/NATS-Bench#47]
I am not sure if there is val/test part in ImageNet16120 dataset. I only found files of train set and val set in document you provided. 
But in [Readme document](https://github.com/D-X-Y/NATS-Bench), the test set mentioned is actually val set, according to your [source code](https://github.com/D-X-Y/AutoDL-Projects/blob/f46486e21b71ae6459a700be720d7648b5429569/xautodl/datasets/DownsampledImageNet.py). 

If the val set is used for test or just wrongly named, did you use the whole training set for model training without spliting?

**Comment by D-X-Y** [comment]
For the original `ImageNet16` and `ImageNet16-120`, the raw dataset only has train/test sets.
So that in the NATS-Bench, I split the original `test` set of `ImageNet16-120` into two parts serving as the new `test` and `val` sets.
